### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by t-bello7**
- Run a program using the command line.
- Set up associations between classes and objects.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```
Comment (brytebee) [inline classroom.rb:11]: - [ ] **Required:** Kindly set the classroom for the student after pushing the input student into the @students instance variable as required. Consider the code snippet below:
```
student.classroom = self
```

File: student.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 require_relative './person'
 
 class Studet < Person
+  attr_accessor :classroom
```
Comment (brytebee) [inline student.rb:4]: - [ ] **Required:** Update this accessor attribute to become only a getter. The setter is defined as the snippet below:

```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
```
Comment (brytebee) [inline rental.rb:8]: - [ ] **Required:** The associations should be properly setup to ensure the relationship between models/classes. Consider the code snippets below:

```
book.rentals << self
person.rentals << self
```

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @t-bello7👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @t-bello7 , 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**